base/special/exp.jl

```diff
@@ -0,0 +1,137 @@
+# Based on FreeBSD lib/msun/src/e_exp.c
+# ====================================================
+# Copyright (C) 2004 by Sun Microsystems, Inc. All rights reserved. Permission
```

Comment:
though I would actually put it in the "The following components of Julia's standard library have separate licenses:" section since it's translated Julia code rather than directly copied C code